src/controllers/controllerengine.cpp

```diff
@@ -684,7 +683,7 @@ double ControllerEngine::getValue(const QString& group, const QString& name) {
    Output:  -
    -------- ------------------------------------------------------ */
 void ControllerEngine::setValue(const QString& group, const QString& name, double newValue) {
-    if (isnan(newValue)) {
+    if (util_isnan(newValue)) {
```

Comment:
`mixxx::math::isnan` would probably look nicer, but this can be taken care of later on if you don't want to do it here.

Reply:
The `util_` prefix is easy to find and replace if we want to move them into a separate namespace. This might be combined with reorganizing the files.